apps/ex_wire/lib/ex_wire/sync.ex

```diff
@@ -38,6 +38,11 @@ defmodule ExWire.Sync do
           last_requested_block: integer() | nil
         }
 
+  @spec get_state() :: state
```

Comment:
hm... did you rebase? I think I added that, no? 
https://github.com/mana-ethereum/mana/blob/master/apps/ex_wire/lib/ex_wire/sync.ex#L49

Reply:
Your branch wasn't merged at the time of writing, so I copied your changes until they were merged. Will rebase and make sure it merges properly.